pkg/web/api_test.go

```diff
@@ -91,23 +91,43 @@ var _ = Describe("API", func() {
 				names := filterNames()
 				Expect(names).To(Equal([]string{
 					"before-existing-plugin:FetchCatalog",
-					"existing-plugin:FetchCatalog",
 					"before-existing-plugin:FetchService",
-					"existing-plugin:FetchService",
 					"before-existing-plugin:Provision",
-					"existing-plugin:Provision",
 					"before-existing-plugin:UpdateService",
-					"existing-plugin:UpdateService",
 					"before-existing-plugin:Deprovision",
-					"existing-plugin:Deprovision",
 					"before-existing-plugin:FetchBinding",
-					"existing-plugin:FetchBinding",
 					"before-existing-plugin:Bind",
-					"existing-plugin:Bind",
 					"before-existing-plugin:Unbind",
+					"existing-plugin:FetchCatalog",
+					"existing-plugin:FetchService",
+					"existing-plugin:Provision",
+					"existing-plugin:UpdateService",
+					"existing-plugin:Deprovision",
+					"existing-plugin:FetchBinding",
+					"existing-plugin:Bind",
 					"existing-plugin:Unbind",
 				}))
 			})
+
+			When("the existing plugin does not implement all plugin interfaces", func() {
```

Comment:
as far as i understand we fix the scenario with "transitive relativity" in which the middle plugin is missing a method X that the first and the third plugin have and because of that the third plugin method X will run before the first plugin method X. Does it make sense to add an actual test for that?